zebra-chain/src/block.rs

```diff
@@ -21,6 +21,7 @@ pub use header::BlockTimeError;
 pub use header::{CountedHeader, Header};
 pub use height::Height;
 pub use root_hash::RootHash;
+pub use serialize::MAX_BLOCK_BYTES;
```

Comment:
I am unsure why this was added if it is not used.
```suggestion
pub use serialize::MAX_BLOCK_BYTES;
```
```suggestion
```

Reply:
That was my mistake from an earlier revision